extensions/mssql/package.json

```diff
@@ -404,6 +452,26 @@
           "when": "connectionProvider == MSSQL && nodeType == Folder && objectType == Tables",
           "group": "0_query@1"
         },
+        {
+          "command": "mssql.newLogin",
+          "when": "connectionProvider == MSSQL && nodeType == Folder && objectType == ServerLevelLogins && config.workbench.enablePreviewFeatures",
+          "group": "0_query@1"
+        },
+        {
+          "command": "mssql.newUser",
+          "when": "connectionProvider == MSSQL && nodeType == Folder && objectType == Users && config.workbench.enablePreviewFeatures",
+          "group": "0_query@1"
+        },
+        {
+          "command": "mssql.objectProperties",
```

Comment:
We'll probably want to make the Admin Tools extension not show a duplicate properties menu item - although given how much more complex the UI is there maybe we should keep it but just rename it to make it clear it's not the "default" properties item

Reply:
I thought about it and reverted my changes in the ssms-min extension, until we have completed all the features in Login/User, I will keep both present, I might be a little bit confusing but user should be able to figure out the differences easily, plus the menu item group is different.